recipes/wpebackend-fdo.recipe

```diff
@@ -1,14 +1,15 @@
 # -*- Mode: Python -*- vi:si:et:sw=4:sts=4:ts=4:syntax=python
 
+import os
 from cerbero.utils import android
 
 class Recipe(recipe.Recipe):
     name = 'wpebackend-fdo'
-    version = '1.8.0'
-    stype = SourceType.GIT
+    version = '1.14.0'
+    stype = SourceType.TARBALL
     btype = BuildType.CMAKE
-    remotes = {'origin': 'https://github.com/Igalia/WPEBackend-fdo.git'}
-    commit = '5523f39d46d398608544a501c39b6f1c3d41233f'
+    url = 'https://wpewebkit.org/releases/wpebackend-fdo-{0}.tar.xz'.format(version)
```

Comment:
not sure why we have this recipe. We are not using this

Reply:
:+1: removed.